althea_kernel_interface/src/exit_client_tunnel.rs

```diff
@@ -58,7 +58,7 @@ impl KernelInterface {
             ],
         )?;
 
-        let prev_ip = self.get_global_device_ip("wg_exit");
+        let prev_ip: Result<Ipv4Addr, Error> = self.get_global_device_ip_v4("wg_exit");
```

Comment:
Rust should be able to infer type here too AFAIK.

Reply:
yes I did that for the sake of making sure I don't screw it up again.